paasta_tools/cli/cmds/status.py

```diff
@@ -785,9 +784,8 @@ def print_flink_status(
         output.append(f"    No other information available in non-running state")
         return 0
 
-    dashboard_url = get_dashboard_url(
-        cluster=cluster, service=service, instance=instance
-    )
+    dashboard_url = metadata.annotations.get("yelp.com/dashboard_url")
```

Comment:
are you missing a `/` separator here?

Reply:
No - the dashboard_url is guaranteed to end with `/`, see `setup_kubernetes_cr.get_dashboard_url` where this field is set.